asdf/extern/atomicfile.py

```diff
@@ -71,13 +71,13 @@ def atomic_rename(src, dst):
         except OSError as e:
             if e.errno != errno.EEXIST:
                 raise
-            old = "%s-%08x" % (dst, random.randint(0, sys.maxsize))
+            old = "%s-%08x" % (dst, random.randint(0, sys.maxsize))  # nosec
             os.rename(dst, old)
             os.rename(src, dst)
             try:
                 os.unlink(old)
             except Exception:
-                pass
+                print(f"'os.unlink({old})' failed")
```

Comment:
We shouldn't be doing any `bandit` checks on the `extern` code.  Use the tox env already set up for bandit to do this check, as it already has the paths that need to be skipped.